config/homs_configuration.default.yml

```diff
@@ -31,6 +31,7 @@ SSO:
     redirect_uri: "http://localhost:4000/authenticate_by_keycloak"
     secret: "af9504fc-b030-405e-97b6-813220c07a7e"
     logout_redirect: "http://localhost:4000"
+    scope: homs_scope
```

Comment:
it should probably be a list of strings, and I think just `homs` for HOMS scope would be enough. No need to type `_scope` suffix for every scope.